forc-plugins/forc-client/src/op/deploy.rs

```diff
@@ -113,6 +121,54 @@ fn validate_and_parse_salts<'a>(
     Ok(contract_salt_map)
 }
 
+async fn deploy_new_proxy(
+    pkg: &BuiltPackage,
+    owner_account_address: &mut Bech32Address,
+    impl_contract: &DeployedContract,
+    build_opts: &BuildOpts,
+    command: &cmd::Deploy,
+    salt: Salt,
+    wallet_mode: &WalletSelectionMode,
+) -> Result<DeployedContract> {
+    println!("  {} proxy contract", "Creating".bold().green());
```

Comment:
```suggestion
    info!("  {} proxy contract", "Creating".bold().green());
```

Perhaps we should also have it say `Creating proxy contract at ~/.forc/.generated_proxy_contracts/path_to_contract`